workers/data_refinery_workers/processors/test_compendia.py

```diff
@@ -71,81 +63,74 @@ def test_create_compendia(self):
+        job = ProcessorJob()
+        job.pipeline_applied = ProcessorPipeline.CREATE_COMPENDIA.value
+        job.save()
+
+        gallus_gallus = Organism.get_object_for_name("GALLUS_GALLUS", taxonomy_id=1001)
+
+        # MICROARRAY TECH
+        (experiment, _) = Experiment.objects.get_or_create(accession_code="GSE1487313")
+        experiment.accession_code = "GSE1487313"
+        experiment.save()
+
+        create_sample_for_experiment(
+            {
+                "organism": gallus_gallus,
+                "accession_code": "GSM1487313",
+                "technology": "MICROARRAY",
+                "filename": "GSM1487313_liver.PCL",
+                "data_dir": DATA_DIR,
+            },
+            experiment,
+        )
+
+        # Missing sample that will be filtered
+        create_sample_for_experiment(
+            {
+                "organism": gallus_gallus,
+                "accession_code": "GSM1487222",
+                "title": "this sample will be filtered",
+                "technology": "MICROARRAY",
+                "filename": "GSM1487222_empty.PCL",
+                "data_dir": DATA_DIR,
+            },
+            experiment,
+        )
 
         # RNASEQ TECH
         experiment2 = Experiment()
         experiment2.accession_code = "SRS332914"
         experiment2.save()
 
-        result2 = ComputationalResult()
-        result2.save()
-
-        sample2 = Sample()
-        sample2.accession_code = "SRS332914"
-        sample2.title = "SRS332914"
-        sample2.organism = gallus_gallus
-        sample2.technology = "RNA-SEQ"
-        sample2.save()
-
-        sra2 = SampleResultAssociation()
-        sra2.sample = sample2
-        sra2.result = result2
-        sra2.save()
-
-        esa2 = ExperimentSampleAssociation()
-        esa2.experiment = experiment2
-        esa2.sample = sample2
-        esa2.save()
-
-        computed_file2 = ComputedFile()
-        computed_file2.filename = "SRP149598_gene_lengthScaledTPM.tsv"
-        computed_file2.absolute_file_path = "/home/user/data_store/PCL/" + computed_file2.filename
-        computed_file2.result = result2
-        computed_file2.size_in_bytes = 234
-        computed_file2.is_smashable = True
-        computed_file2.save()
-
-        assoc2 = SampleComputedFileAssociation()
-        assoc2.sample = sample2
-        assoc2.computed_file = computed_file2
-        assoc2.save()
+        create_sample_for_experiment(
+            {
+                "organism": gallus_gallus,
+                "accession_code": "SRS332914",
+                "technology": "RNA-SEQ",
+                "filename": "SRP149598_gene_lengthScaledTPM.tsv",
+                "data_dir": DATA_DIR,
+            },
+            experiment,
+        )
 
         dset = Dataset()
-        dset.data = {"GSE1487313": ["GSM1487313", "GSM1487222"], "SRX332914": ["SRS332914"]}
+        dset.data = {
+            "GSE1487313": ["GSM1487313", "GSM1487222"],
+            "SRX332914": ["SRS332914"],
```

Comment:
Is this supposed to be an SRX? I know it was before... but that's kinda confusing.